acceptance/ui/features/support/selenium.rb

```diff
@@ -0,0 +1,22 @@
+
+#
+# Modifications to the selenium behavior for our tests.
+#
+
+class Capybara::Selenium::Driver
```

Comment:
By default, we use the `selenium_chrome` driver instead of the `selenium` driver  (i.e test against Chrome instead of Firefox). Does this change work for both browsers, or just one?